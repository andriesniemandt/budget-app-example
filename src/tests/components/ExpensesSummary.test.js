import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('Should correctly render expenses summary with one expense', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={235} />);
    expect(wrapper).toMatchSnapshot();
});

test('Should correctly render expenses summary with multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={23} expensesTotal={235124412421} />);
    expect(wrapper).toMatchSnapshot();
});