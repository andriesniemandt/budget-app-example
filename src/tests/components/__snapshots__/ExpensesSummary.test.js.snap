// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should correctly render expenses summary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $2,351,244,124.21
  </h1>
</div>
`;

exports[`Should correctly render expenses summary with one expense 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $2.35
  </h1>
</div>
`;
